Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
```

Comment:
Let's remove `Pipefile` & its lock file as we won't be using them. We're using pip + setuptools + virtualenv so far and might move towards poetry later.

All the dependencies are centralized here already - https://github.com/leapfrogtechnology/oss-leaderboard/blob/master/setup.py.